crypto/BRCryptoTransfer.c

```diff
@@ -416,6 +420,27 @@ cryptoTransferGetUnitForFee (BRCryptoTransfer transfer) {
     return cryptoUnitTake (transfer->unitForFee);
 }
 
+extern size_t
+cryptoTransferGetAttributeCount (BRCryptoTransfer transfer) {
+    return array_count(transfer->attributes);
```

Comment:
There will be thread-sanitizer warnings here.

Reply:
Fixed.  I don't see the BRCryptoWallet additions as needing a lock.